packages/fireworks/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "./node_modules/@vercel/ai-tsconfig/react-library.json",
```

Comment:
not related to this pr specifically, but i'm wondering what we should ideally extend for providers (since they are not react based)